src/main/java/uk/gov/hmcts/reform/ucmc/controllers/CreateClaimController.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.ucmc.controllers;
+
+import io.swagger.annotations.Api;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.SubmittedCallbackResponse;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import static java.lang.String.format;
+import static uk.gov.hmcts.reform.ucmc.helpers.DateFormatHelper.DATE_TIME_AT;
+import static uk.gov.hmcts.reform.ucmc.helpers.DateFormatHelper.formatLocalDateTime;
+
+@Api
+@RestController
+@RequestMapping("/create-claim")
+public class CreateClaimController {
+
+    @PostMapping("/submitted")
+    public SubmittedCallbackResponse handleSubmitted() {
+        String documentLink = "https://www.google.com";
+        String responsePackLink = "https://formfinder.hmctsformfinder.justice.gov.uk/n9-eng.pdf";
```

Comment:
If this is going to be static link, would be nice to have this configurable.

Reply:
do you see value in moving this to a constants file?

Reply:
If that changes any time we can update the configurations to override this from our yaml file instead of rebuilding & redeploying the application.